@here/harp-mapview/lib/Tile.ts

```diff
@@ -664,7 +669,7 @@ export class Tile implements CachedResource {
      * @note Negative values indicates depressions.
      */
     get maxElevation(): number {
-        return this.m_maxElevation;
+        return this.m_maxElevation + this.maxGeometryHeight;
```

Comment:
I think we may not leave it inconsistent, some new getter should be added, i.e. get maxHeight, otherwise simple unit test fails:
```
tile.maxElevation = 100;
assert_equal(tile.maxElevation, 100);
```
and more importantly it becomes misleading.

Reply:
fixed